packages/expo-maps/src/apple/AppleMaps.types.ts

```diff
@@ -116,6 +139,25 @@ export type AppleMapsAnnotation = {
   icon?: SharedRefType<'image'>;
 } & AppleMapsMarker;
 
+export type AppleMapsPolyline = {
+  /**
+   * The coordinates of the polyline.
+   */
+  coordinates: Coordinates[];
+  /**
+   * The color of the polyline.
+   */
+  strokeColor?: ProcessedColorValue | string | null;
+  /**
+   * The width of the polyline.
+   */
+  strokeWidth?: number;
```

Comment:
```suggestion
  color?: ProcessedColorValue | string | null;
  /**
   * The width of the polyline.
   */
  width?: number;
```
I think we can use the same names on both platforms.

Reply:
Done